src/game/object/creature.h

```diff
@@ -105,6 +105,7 @@ class Creature : public SpatialObject {
     void setTalking(bool talking);
     void setFaction(Faction faction);
     void setMovementRestricted(bool restricted);
+    void setCombat(bool active) { _inCombat = active; }
```

Comment:
nit: This is a setter function for a particular member variable. Maybe name it "setInCombat"?